applications/matter_weather_station/README.rst

```diff
@@ -181,6 +199,29 @@ The ``build_thingy53_nrf5340_cpuapp`` parameter specifies the output directory f
 
       File not found: ./ncs/nrf/applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/prj_shell.conf
 
+Building for the nRF7002 Wi-Fi extension board
+==============================================
+
+To build this application to work with the nRF7002 Wi-Fi extension board:
+
+1. Connect the nRF7002 EB to the **P9** connector on Thingy:53.
+#. Build the application:
+
+   .. tabs::
+
+      .. group-tab:: nRF Connect for VS Code
+
+         To build the application in the nRF Connect for VS Code IDE for Thingy:53 with the nRF7002 EB attached, add the extension board and the build type variables in the build configuration's :guilabel:`Extra CMake arguments` and rebuild the build configuration.
+         For example: ``-- -DSHIELD=nrf21540_ek -DCONF_FILE=prj_release.conf``.
```

Comment:
nRF21540?

Reply:
Thanks for fixing this, @ArekBalysNordic